### PR TITLE
add gitbook cn version of user guide 

### PR DESCRIPTION
**Pull request by imlidian**

### CODE REVIEW

**Comment by imlidian** [comment]
you can test with this gitbook https://imlidian.github.io/cn/users/

File: _data/navigation.yml [hunk 1]
```diff
@@ -204,96 +206,24 @@ t:
         url: /cn/docs/quick-start-dataconsistency/
 
     users:
-      - title: 用户手册
+      - title: Java Chassis
         children:
           - title: Java Chassis系统架构
             url: /cn/users/
           - title: 环境配置
```
Comment (zenlinTechnofreak) [inline _data/navigation.yml]: You reserve the environment configuration, and is this part havs been included in the quickstart?  if not, how about move this part to  the quickstart.

File: _data/navigation.yml [hunk 2]
```diff
@@ -204,96 +206,24 @@ t:
         url: /cn/docs/quick-start-dataconsistency/
 
     users:
-      - title: 用户手册
+      - title: Java Chassis
         children:
           - title: Java Chassis系统架构
```
Comment (zenlinTechnofreak) [inline _data/navigation.yml]: I guess you just want to keep a section to briefly introduce the sub projects of servicecomb, but not a “java-chassis architecture”, right?

File: _users/cn/servicecomb-java-chassis-doc/zh_CN/introduction.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+title: "Java Chassis"
+lang: cn
+ref: servicecomb-java-chassis-doc
+permalink: /cn/users/servicecomb-java-chassis-doc/zh_CN/introduction/
+excerpt: "Java Chassis"
+last_modified_at: 2017-08-21T15:01:43-04:00
+redirect_from:
+  - /theme-setup/
+---
+
```
Comment (zenlinTechnofreak) [inline _users/cn/servicecomb-java-chassis-doc/zh_CN/introduction.md]: Could you give a title for ther user book, like "Java-Chassis 用户手册" ?  No book mark here after jumped to user book.

**Comment by zenlinTechnofreak** [comment]
@imlidian I have give some suggestions using the Single line command, please help to deal with it. Thanks.

File: _data/navigation.yml (hunk 2 above)
Comment (imlidian) [reply _data/navigation.yml]: Right. My original idea was to keep the same style as before.

File: _users/cn/servicecomb-java-chassis-doc/zh_CN/introduction.md (hunk 3 above)
Comment (imlidian) [reply _users/cn/servicecomb-java-chassis-doc/zh_CN/introduction.md]: Agree with you. Already git it done

**Comment by zenlinTechnofreak** [comment]
@imlidian You said you agree with me, and some of them have been done, but I have not seen the corresponding changes?

**Comment by WillemJiang** [comment]
There are too much changes here, we just need to updated the content directory in asf-site for the new released the git doc page.

**Comment by imlidian** [comment]
@WillemJiang base on the last update. we only update navigation.yml and add script dir for all function work. so no more changes already.

**Comment by WillemJiang** [comment]
@imlidian  Can you squash the commit,  there are about 13 commits here.

**Comment by imlidian** [comment]
@WillemJiang done. now, there are only 2 commits.

File: _data/navigation.yml [hunk 4]
```diff
@@ -204,96 +210,20 @@ t:
...
 
-      - title: 部署
+      - title: Saga
         children:
-          - title: 运行模式
-            url: /cn/users/run-mode/
-          - title: 部署模式
-            url: /cn/users/deploy-mode/
+          - title: Saga用户手册
+            url: https://github.com/apache/incubator-servicecomb-saga
```
Comment (WillemJiang) [inline _data/navigation.yml]: We need to move the saga user document doc repo.

Comment (WillemJiang) [reply _data/navigation.yml]: It can be done in the next PR.

**Pull request merged**